RevenueCatUI/PaywallView.swift

```diff
@@ -33,19 +33,29 @@ public struct PaywallView: View {
     @Environment(\.locale)
     private var locale
 
+    // Do NOT reference this object directly, always
+    // use `purchaseHandler`. It is here for lifecycle
+    // purposes only and may or may not be the same
+    // object as `purchaseHandler`.
```

Comment:
Could you explain in this comment what is meant by lifecycle purposes? I know I will totally forget what this means in like a week 😅

Reply:
Update the comment to:
```
    // Do NOT reference this object directly, always use `purchaseHandler`.
    // `doNotAccessPurchaseHandler` is here to own the `PurchaseHandler` when
    //  it is owned by this view. When it is owned by an external object, this
    //  references a "dummy" purchase handler.
```